flake.nix

```diff
@@ -8,7 +8,7 @@
       # see https://github.com/renovatebot/renovate/issues/29721 and
       # https://github.com/renovatebot/renovate/blob/743fed0ec6ca5810e274571c83fa6d4f5213d4e7/lib/modules/manager/nix/extract.ts#L6.
       # We must keep the following string in the file for renovate to work: "github:NixOS/nixpkgs/nixpkgs-unstable"
-      url = "github:NixOS/nixpkgs?ref=nixos-unstable";
+      url = "github:msanft/nixpkgs/msanft/libnvidia-container/driver-bins";
```

Comment:
The comment above does not apply anymore, and it would be good to have a note that says how we get back to unstable.

Reply:
This commit is to be removed, see the message